$NetBSD: patch-ad,v 1.6 2006/11/24 16:31:48 rillig Exp $

The second hunk is needed for Solaris with SunPro, but I don't have a
clue why.

See also:
* http://src.opensolaris.org/source/xref/jds/spec-files/trunk/patches/aspell-01-forte.diff

--- common/convert.cpp.orig	2006-09-17 19:24:05.000000000 +0200
+++ common/convert.cpp	2006-09-17 19:24:46.000000000 +0200
@@ -24,6 +24,10 @@
 
 #include "gettext.h"
 
+#ifdef DECLARE_SNPRINTF
+int snprintf(char *str, size_t size, const char *format, ...);
+#endif
+
 namespace acommon {
 
   typedef unsigned char  byte;
@@ -238,7 +242,7 @@ namespace acommon {
   }
 
   template <class T>
-  static void free_norm_table(NormTable<T> * d)
+  void free_norm_table(NormTable<T> * d)
   {
     for (T * cur = d->data; cur != d->end; ++cur) {
       if (cur->sub_table) 
